projects/admin/src/app/record/detail-view/item-detail-view/item-transaction/item-transaction.component.ts

```diff
@@ -141,7 +139,8 @@ export class ItemTransactionComponent implements OnInit, OnDestroy {
     // No permission API in backend
     // Allowed when loan state is PENDING or ITEM_IN_TRANSIT_FOR_PICKUP according to actions.md
     return this.transaction.metadata.state === LoanState.PENDING
-    || this.transaction.metadata.state === LoanState.ITEM_IN_TRANSIT_FOR_PICKUP;
+    || this.transaction.metadata.state === LoanState.ITEM_IN_TRANSIT_FOR_PICKUP
```

Comment:
You can probably replace by some: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/some

Reply:
Done